Submitted By: Anthony Creech (lunatic at qx dot net)
Date: 10/23/12
Initial Package Version: 1.4.16
Origin: Gentoo
Upstream Status: Accepted
Description: stdio: don't assume gets any more 
                    hack until m4 pulls a newer gnulib version

--- a/lib/stdio.in.h
+++ b/lib/stdio.in.h
@@ -125,7 +125,6 @@
    so any use of gets warrants an unconditional warning.  Assume it is
    always declared, since it is required by C89.  */
 #undef gets
-_GL_WARN_ON_USE (gets, "gets is a security hole - use fgets instead");
 
 #if @GNULIB_FOPEN@
 # if @REPLACE_FOPEN@
